README.md

```diff
@@ -57,9 +65,29 @@ app.register_function(sample_page)
 # Run the server
 server = app.run()
 ```
+## Whos is PyCob for?
+
+PyCob is perfect for quant and analytical developers who want to expose data analysis.  PyCob's sweetspot sits between jupyter notebooks and apps that are supported by a full frontend dev team.  
+
+
+### Scaling an app
+#### Jupyter notebook on your machine
+build whatever you want, rearrange cells, try new stuff.  Dont expect anyone else to run or understand your code
+#### Jupyter notebooks shared with other data scientists
+Put your notebook into a git repo, add a `conda.yml` or `requirements.txt` get feedback on the algorithm
+This works for 1-4 technically proficient data scientists and engineers
+
+To scale beyond this step you're going to need to do something different
+#### Stick with Jupyter notebooks
+If you want to build an app in the jupyter notebook environment, you are going to need to buy into some infrastructure.  At this point you will want a jupyter hosting system like Domino ($100k/year) SaturnCloud ($10-20k/year) or hex (??? $20k+ / year).  This will work for 5-50 users
+#### PyCob
+Integrate your app into PyCob, and setup a fairly standard python hosting (please expand)
+
+works for 5-100 non-technical users
 
-### Quickstart
-[Quickstart Template Repo](https://github.com/pycob/quickstart)
```

Comment:
I think you should have this all in a single repo, at least initially.  Your early adopters will be more technical and will likely want to modify PyCob itself.  Having multiple repos is a distraction.  I would put your example demos into this repo.